controller/TransparencyController.js

```diff
@@ -0,0 +1,50 @@
+const { Donation } = require("../services");
+const catchAsyncError = require("../middleware/catchAsyncError");
+
+/**
+ * @description get total donations
+ * @route GET /v1/transparency
+ * @access PUBLIC
+ */
+exports.getTotalAmount = catchAsyncError(async (req, res) => {
+  const startDate = new Date(req.query.startDate);
+  const endDate = new Date(req.query.endDate);
+
+  const totalDonations = await Donation.getTotal(startDate, endDate);
+
+  return res.status(200).json({
+    status: true,
+    data: totalDonations
+  });
+});
+
+/**
+ * @description get total donors and donations
+ * @route GET /v1/transparency
```

Comment:
Please update this endpoint to `/v1/donations/overview` because this is essentially an overview of the total donation amount and number of donors.

Reply:
Done.